.github/workflows/check_compilable_percentage.yml

```diff
@@ -108,37 +109,36 @@ jobs:
       - name: Show only failed minimizations
         run: |
           grep 'failed_minimization' ASHE/logs/specimin_run_results.txt | sort
+          grep 'failed_minimization' ASHE/logs/specimin_run_results.txt | sort > run_failed_minimizations.txt
       - name: Show only failed compilations
         run: |
           grep 'failed_compilation' ASHE/logs/specimin_run_results.txt | sort
-      - name: Parse accuracy percentage
-        id: parse_accuracy_percentage
+          grep 'failed_compilation' ASHE/logs/specimin_run_results.txt | sort > run_failed_compilations.txt
```

Comment:
Use `tee` (https://en.wikipedia.org/wiki/Tee_(command)) instead of running this command twice.